### ISSUE TITLE
aria-hidden="true" never removed from root element after closing abstract modal

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-framework#842]
# Bug Report

## Description
When opening the abstract modal, we add `aria-hidden="true"` and and `inert` to the "root" element. When the abstract modal is closed, we should be removing these. The `inert` attribute is successfully removed, but the `aria-hidden="true"` is never removed or set to false. This causes issues with screen readers as the entire app appears to be hidden after a modal is closed.

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-framework/components/terra-abstract-modal/abstract-modal/abstract-modal
2. Inspect the DOM and find div#root
3. Open the default abstract modal
4. Close the default abstract modal
5. See that `aria-hidden="true"` is still set on div#root.

## Additional Context / Screenshots
I think this happened sometime between yesterdays release and the prior framework release on July 30, 2019.

This seems to happen on both desktop and mobile.

## Expected Behavior
`aria-hidden="true"` is removed from the #root element when the abstract modal is closed

## Environment

* Component Name and Version: abstract-modal v3.8.0
* Browser Name and Version: Chrome Latest
* Operating System and version (desktop or mobile): Mac OS / iOS
